src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -879,6 +880,8 @@ resultMetadata.put("content.type","image/png")
 
 A Task result can be added to the list of results of a Job, see section <<_result_list>> below.
 
+NOTE: Specific result metadata described above can also be set using <<_generic_information,Generic Information>>. For example, defining the generic information _content.type = text/html_ is equivalent to adding this value to the *resultMetadata* map.
```

Comment:
'can be also set' instead of 'can also be set'

Reply:
ok

Reply:
actually "can also be set" is more english